tools/cmd/doc-gen/main.go

```diff
@@ -32,7 +32,7 @@ import (
 	devopsv1alpha2 "kubesphere.io/kubesphere/pkg/kapis/devops/v1alpha2"
 	iamv1alpha2 "kubesphere.io/kubesphere/pkg/kapis/iam/v1alpha2"
 	loggingv1alpha2 "kubesphere.io/kubesphere/pkg/kapis/logging/v1alpha2"
-	monitoringv1alpha2 "kubesphere.io/kubesphere/pkg/kapis/monitoring/v1alpha2"
+	monitoringv1alpha2 "kubesphere.io/kubesphere/pkg/kapis/monitoring/v1alpha3"
```

Comment:
The same problem here: monitoringv1alpha2  doesn't match "kubesphere.io/kubesphere/pkg/kapis/monitoring/v1alpha3"

Reply:
done